components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -116,6 +118,19 @@ const BillingSubscriptions: React.FC = () => {
         return null;
     }
 
+    const shouldShowGrandfatheredPlanBanner = () => {
+        // Give preference to the payment failed banner
+        return (
+            !shouldShowPaymentFailedBanner() &&
+            showGrandfatheredPlanBanner &&
+
+            // This banner is only for 1 specific grandfathered subscription type. The below ID's are for test and production
+            ['prod_HyiHEAVKW5bYG3', 'prod_Hm2oYaBiRSISL2'].includes(
+                subscription?.product_id,
+            )
```

Comment:
Would it be possible to rely on [`CloudProducts.LEGACY`](https://github.com/mattermost/mattermost-webapp/blob/d87786b20ae37013ddcc0fdaa84542ca0175a9c7/utils/constants.tsx#L411)?

Reply:
The SKU isn't stored on the subscription, only the product. So we'd need extra lookups for those, then filters to find the correct product, then check the sku, etc. For such a small banner I'm not sure its worth it, but if you feel strongly about it lmk

Reply:
ditto not sure its worth it. As an informational FYI, we added a selector for getting the subscription product since its repeated in lots of places: https://github.com/mattermost/mattermost-webapp/blob/master/packages/mattermost-redux/src/selectors/entities/cloud.ts#L25

Reply:
Don't feel strongly about it and it will be short-lived. Let's have a ticket to remove this after Nov 1